futaba/sql/models/roles.py

```diff
@@ -151,12 +152,14 @@ def get_pingable_role_channels(self, guild):
         for (channel_id, role_id) in result.fetchall():
             channel = discord.utils.get(guild.channels, id=channel_id)
             role = discord.utils.get(guild.roles, id=role_id)
-            if role is not None:
+            if not role:
+                self.remove_nonexistent_role_channel(guild, channel_id, role_id)
+            else:
```

Comment:
If you have an if-else branch I would prefer not having the inverted condition first

Reply:
Done.